glide.yaml

```diff
@@ -29,7 +29,7 @@ import:
 - package: github.com/uber-go/tally
   version: ^3
 - package: github.com/julienschmidt/httprouter
-  version: v1.1
+  version: ^1.1
```

Comment:
This is a noop.

Reply:
I was hoping there would be a semver release in the future. Updated to master anyway.